src/main.cpp

```diff
@@ -170,14 +176,22 @@ int main()
             // looks good up to this point
 
             // std::cout << "State vector after " << i << " time steps: [" << lm.getStateVector()->data[0] << ", " << lm.getStateVector()->data[1] << "]" << std::endl;
+			
```

Comment:
since we are runnning a loop here. How will that affect speed of our code @DiligentBuilder ??

Reply:
May be we can try something like memcpy which might be faster but u have to analyze if it really is faster.
    #include <cstring> // For memcpy
    int source[5] = {1, 2, 3, 4, 5};
    int destination[5];
    memcpy(destination, source, sizeof(source));